mapie/control_risk/p_values.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Iterable, Union
+from scipy.stats import binom
+
+
+def hoefdding_bentkus_p_value(
+    r_hat: NDArray,
+    n: int,
+    alpha: Union[float, NDArray]
+) -> NDArray:
+    """
+    Parameters
+    ----------
+    r_hat : NDArray of shape (len(lambdas), )
```

Comment:
ERROR: lambdas is not defined. You should take another reference (`n_samples` or `n_params`)

Reply:
Done

Reply:
Done